source/gui/guiHelper.py

```diff
@@ -458,3 +462,55 @@ class SIPABCMeta(wx.siplib.wrappertype, ABCMeta):
 	"""Meta class to be used for wx subclasses with abstract methods."""
 
 	pass
+
+
+class _WxCallOnMainResult:
+	"""Container to hold either the return value or exception raised by a function."""
+
+	__slots__ = ("result", "exception")
```

Comment:
TIL about slots!
Can you add[ type annotations?](https://stackoverflow.com/a/52553217)

```suggestion
	__slots__ = ("result", "exception")
	
	result: Any
	exception: Exception
```